__tests__/app.test.js

```diff
@@ -412,3 +413,25 @@ describe("POST/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: deletes the specified comment from the data and does not return any content", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then((results) => {
+        expect(results.text).toBe("");
```

Comment:
Dont need this bit as 204 wont return content

Reply:
Could do a query to check the db for the thing thats been deleted if you wanted something to put here